sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryImageUpdate.cs

```diff
@@ -48,22 +48,23 @@ public GalleryImageUpdate()
-        /// Definition resource. This property is updatable.</param>
-        /// <param name="eula">The Eula agreement for the gallery Image
-        /// Definition.</param>
+        /// <param name="description">The description of this gallery image
+        /// definition resource. This property is updatable.</param>
+        /// <param name="eula">The Eula agreement for the gallery image
+        /// definition.</param>
         /// <param name="privacyStatementUri">The privacy statement
         /// uri.</param>
         /// <param name="releaseNoteUri">The release note uri.</param>
         /// <param name="hyperVGeneration">The hypervisor generation of the
         /// Virtual Machine. Applicable to OS disks only. Possible values
         /// include: 'V1', 'V2'</param>
+        /// <param name="features">A list of gallery image features.</param>
         /// <param name="endOfLifeDate">The end of life date of the gallery
-        /// Image Definition. This property can be used for decommissioning
+        /// image definition. This property can be used for decommissioning
         /// purposes. This property is updatable.</param>
         /// <param name="provisioningState">The current state of the gallery
-        /// Image Definition.</param>
-        public GalleryImageUpdate(OperatingSystemTypes osType, OperatingSystemStateTypes osState, GalleryImageIdentifier identifier, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string description = default(string), string eula = default(string), string privacyStatementUri = default(string), string releaseNoteUri = default(string), string hyperVGeneration = default(string), System.DateTime? endOfLifeDate = default(System.DateTime?), RecommendedMachineConfiguration recommended = default(RecommendedMachineConfiguration), Disallowed disallowed = default(Disallowed), ImagePurchasePlan purchasePlan = default(ImagePurchasePlan), string provisioningState = default(string))
+        /// image definition.</param>
+        public GalleryImageUpdate(OperatingSystemTypes osType, OperatingSystemStateTypes osState, GalleryImageIdentifier identifier, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string description = default(string), string eula = default(string), string privacyStatementUri = default(string), string releaseNoteUri = default(string), string hyperVGeneration = default(string), IList<GalleryImageFeature> features = default(IList<GalleryImageFeature>), System.DateTime? endOfLifeDate = default(System.DateTime?), RecommendedMachineConfiguration recommended = default(RecommendedMachineConfiguration), Disallowed disallowed = default(Disallowed), ImagePurchasePlan purchasePlan = default(ImagePurchasePlan), string provisioningState = default(string))
```

Comment:
This is a breaking change, should add a customization with a constructor overload that matches the old version

Reply:
added